src/content/en/2022/capabilities.md

```diff
@@ -16,7 +16,372 @@ featured_stat_2: TODO
+
+```js
+navigator.mediaSession.metadata = new MediaMetadata({
+  title: "Creep",
+  artist: "Radiohead",
+  album: "Pablo Honey",
+  artwork: [
+    {
+      src: "https://via.placeholder.com/96",
+      sizes: "96x96",
+      type: "image/png",
+    },
+    {
+      src: "https://via.placeholder.com/128",
+      sizes: "128x128",
+      type: "image/png",
+    },
+    {
+      src: "https://via.placeholder.com/192",
+      sizes: "192x192",
+      type: "image/png",
+    },
+    {
+      src: "https://via.placeholder.com/256",
+      sizes: "256x256",
+      type: "image/png",
+    },
+    {
+      src: "https://via.placeholder.com/384",
+      sizes: "384x384",
+      type: "image/png",
+    },
+    {
+      src: "https://via.placeholder.com/512",
+      sizes: "512x512",
+      type: "image/png",
+    },
+  ],
+});
+```
+
+The Media Session API is available on modern versions of Chrome, Edge, Firefox, and Safari.
+
+### Usage of Media Session API
+
+{{ figure_markup(
+  caption="Usage of Media Session API on mobile.",
+  content="7.41%",
+  classes="big-number",
+  sheets_gid="1887140434",
+  sql_file="fugu.sql"
+)
+}}
+
+The Media Session API was not measured last year. In its first year of tracking, the API was used on 8.37% of desktop pages and 7.41% of mobile pages, making it the fourth most used capability on desktop and mobile.
+
+Web applications such as YouTube, YouTube Music, Spotify, and others take advantage of the Media Session API and provide rich controls for the video or audio played.
+
+{{ figure_markup(
+  image="Media-Session-API.png",
+  caption="Accessing controls and information for YouTube Music via the Window's Taskbar.",
+  description="Accessing controls and information for YouTube Music via the Window's Taskbar.",
+) }}
+
+For a deeper dive into video usage on the web, check out the [Media](../media#video) chapter.
+
+## Device Memory API
+
+A device's capabilities depend on a few things, like the network, the CPU core count, and the amount of memory available. The [_Device Memory API_](https://developer.mozilla.org/en-US/docs/Web/API/Device_Memory_API) provides insight into the memory available by providing the read-only property `deviceMemory` on the `Navigator` interface. The property returns an approximate amount of device memory in gigabytes as a floating point number.
+
+The value returned is imprecise, protecting the user's privacy. It's calculated by rounding down the nearest power of 2, then dividing that number by 1,024. The number is also clamped within an upper and lower bound. So you can expect the numbers: `0.25`, `0.5`, `1`, `2`, `4`, and `8` (gigabytes).
+
+```js
+const memory = navigator.deviceMemory;
+console.log('This device has at least ', memory, 'GiB of RAM.');
```

Comment:
Note I had to change this slightly as our code formatter didn't like the other format